data/goruco/goruco-2008/videos.yml

```diff
@@ -0,0 +1,100 @@
+---
+
+# http://2008.goruco.com/agenda/
+
+- title: 'Story Driven Development: The Next Generation of Rails Functional Testing'
+  raw_title: 'Story Driven Development: The Next Generation of Rails Functional Testing by Bryan Helmkamp'
+  speakers:
+    - Bryan Helmkamp
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGkg/
+  video_id: ApD-F65Vv-Q
+
+- title: 'Archaeopteryx: A Ruby MIDI Generator'
+  raw_title: 'Archaeopteryx: A Ruby MIDI Generator by Giles Bowkett'
+  speakers:
+    - Giles Bowkett
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGki/
+  video_id: 0XDawYp9mKY
+
+- title: 'Forbidden Fruit: A Test of Ruby''s Parse Tree'
+  raw_title: 'Forbidden Fruit: A Test of Ruby''s Parse Tree by Chris Wanstrath'
+  speakers:
+    - Chris Wanstrath
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGkk/
+  video_id: DAoYufIKebU
+
+# Lunch
+
+- title: Hurting Code for Fun and Profit
+  raw_title: Hurting Code for Fun and Profit by Ryan Davis
+  speakers:
+    - Ryan Davis
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGkh/
+  video_id: L_z5oqPrDWY
+
+- title: Collective Intelligence
+  raw_title: Collective Intelligence by Paul Dix
+  speakers:
+    - Paul Dix
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGkj/
+  video_id: dNzA5Yygm5U
+
+- title: Merb, All you need, nil you don't
+  raw_title: Goruco 2008 Merb, All you need, nil you don't by Ezra Zygmuntowicz
+  speakers:
+    - Ezra Zygmuntowicz
+  event_name: GoRuCo 2008
+  published_at: '2008-04-26'
+  description: |-
+    Help us caption & translate this video!
+
+    http://amara.org/v/FGkl/
+  video_id: BqjPOC_ckG4
+
+# Break
+
+- title: 'Lightning Talks: RejectConf'
+  raw_title: RejectConf / Lightning Talks by Various Presenters
+  speakers:
+    - Ryan Davis
+    - Johnson
+    - Delmont
+    - Gregory Brown
+    - Dan DiMaggio
+    - Phil Matarese
+    - Peter Jaros
+    # - TODO # Mindstorms
+    - Luke Miglia
+    - Wilson
```

Comment:
I understand that it is sometime complicated for lighting talk to get the full name of the speaker. 
When we just have the first name it means we will create a speaker with just the first name and this could create confusion 

what do you think to comment those speakers and add a comment `missing last name` maybe someone can fix that in the future?

Reply:
Yeah, that's fair 👍🏼